opentargets.json

```diff
@@ -1144,10 +1149,11 @@
               "id": {
                 "$ref": "#/definitions/variantId"
               },
-              "variantFunctionalConsequenceId": {
+              "functionalConsequenceId": {
                 "$ref": "#/definitions/variantFunctionalConsequenceId"
```

Comment:
Question: the name of the field changed. I agree this is a reasonable call, as this field captures different aspects, however the field definition still points to the `#/definitions/variantFunctionalConsequenceId`. So the field has to match `"pattern": "SO_\\d+"` Is it desired?